docs/KF_NOTEBOOK_JG_WORKFLOW.md

```diff
@@ -0,0 +1,11 @@
+# Kids First-Sentieon Joint Cohort Call Notebook
+Due to platform limitations, running the Preferred workflow may not be feasible when cohorts exceed 2200 samples, due to scatter job size limitations.
```

Comment:
Idk if this has an obvious answer, but what should users do if the cohort is > 2200? Should they break up the samples into smaller cohorts and do multiple run